Doc/tutorial/datastructures.rst

```diff
@@ -678,18 +678,19 @@ intended.
 
 Comparing Sequences and Other Types
 ===================================
-
-Sequence objects may be compared to other objects with the same sequence type.
```

Comment:
How about simpler change that doesn't detract from the core message: Sequence objects typically may be compared ..."

That is more technically correct.  The *Sequence* API  does not guarantee comparability but it is common.